GooglePlayInstant/Editor/AssetBundleVerifierWindow.cs

```diff
@@ -39,21 +41,23 @@ public static void ShowWindow()
         {
             // Set AssetBundle url in a private variable so that information displayed in window is consistent with
             // the url that this was called on. 
-            _assetBundleUrl = QuickDeployConfig.Config.assetBundleUrl;
-            UpdateAssetBundleVerificationInfoWindow();
-            GetWindow(typeof(PlayInstantAssetBundleUrlVerifierWindow), true, "Play Instant AssetBundle Verify");
+            var window = (AssetBundleVerifierWindow) GetWindow(typeof(AssetBundleVerifierWindow), true,
+                "Play Instant AssetBundle Verify");
+            window._assetBundleUrl = QuickDeployConfig.Config.assetBundleUrl;
+
+            window.startAssetBundleVerificationDownload();
         }
 
         //TODO: Support Unity 5.6.0+
-        private static void UpdateAssetBundleVerificationInfoWindow()
+        private void startAssetBundleVerificationDownload()
         {
-            var www = UnityWebRequestAssetBundle.GetAssetBundle(_assetBundleUrl);
+            www = UnityWebRequestAssetBundle.GetAssetBundle(_assetBundleUrl);
             www.SendWebRequest();
```

Comment:
Do you also need to set all of the fields back to their default values? E.g.

_errorDescription = null;
 _mainScene = null
etc.

I ask because below you have a "www = null" which tells me you plan to support resending the request.

Reply:
Not wanted behavior because then that would change the GUI's values live, when the user might still be analyzing them. I dispose of the existing _webrequest object /only/ to notify the Update() function to not continuously complete the downloading process.

(Further explanation: Update() starts the download process when _webrequest objects are "done". I then set the _webrequest to null to that Update() doesn't REdownload the bundle again since _webrequest is still done. I still want to GUI to display the values for that download session.)